Library/Formula/unrar.rb

```diff
@@ -15,6 +15,11 @@ class Unrar < Formula
   def install
     system "make"
     bin.install 'unrar'
+    system "make", "clean"
```

Comment:
Why do we need to `clean` afterwards? Is there a `make install` we can try and use or patch instead?